avm/res/container-instance/container-group/main.bicep

```diff
@@ -34,7 +34,7 @@ param imageRegistryCredentials imageRegistryCredentialType[]?
 @description('Optional. Location for all Resources.')
 param location string = resourceGroup().location
 
-@description('Optional. Specify level of protection of the domain name label.')
+@description('Conditional. The value representing the security enum. The \'Unsecure\' value is the default value if not selected and means the object\'s domain name label is not secured against subdomain takeover. The \'TenantReuse\' value is the default value if selected and means the object\'s domain name label can be reused within the same tenant. The \'SubscriptionReuse\' value means the object\'s domain name label can be reused within the same subscription. The \'ResourceGroupReuse\' value means the object\'s domain name label can be reused within the same resource group. The \'NoReuse\' value means the object\'s domain name label cannot be reused within the same resource group, subscription, or tenant. Required if `DnsNameLabel` is set.')
```

Comment:
Suggestion:

Condition for this and for `dnsNameLabel` should probably be "Required if `ipAddressType` is `Public`"

Reply:
ping @JPEasier 😏